sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -3,12 +3,15 @@
 ## 2.0.0-beta.3 (Unreleased)
 
 ### Features Added
+- Added support for a new communication identifier `MicrosoftTeamsAppIdentifier`.
 
 ### Breaking Changes
+- Removal of `MicrosoftBotIdentifier` and adding of `MicrosoftTeamsAppIdentifier` is a breaking change. Identifiers using rawID starting with `28:orgid:` will be treated as `MicrosoftTeamsAppIdentifier`, other rawIDs starting with `28:` will be treated as `UnknownIdentifier`.
 
 ### Bugs Fixed
 
 ### Other Changes
+- Removed `MicrosoftBotIdentifier`.
```

Comment:
you do not have to mention it, as it's introduced for beta version

Reply:
Removed.